broker-node/app/Http/Controllers/UploadSessionController.php

```diff
@@ -132,4 +133,38 @@ public function chunkStatus(Request $request)
 
         return response()->json(['status' => $data_map['status']]);
     }
+
+    public function brokerNodeListener(Request $request)
+    {
+        $cmd = $request->input('command');
+        // This is a hack to cast an associative array to an object.
+        // I don't know how to use PHP properly :(
+        $req = (object)[
+            "command" => $request->input('command'),
+            "message" => $request->input('message'),
+            "chunkId" => $request->input('chunkId'),
+            "address" => "WHLOOOOOOAAAAAAAAAAAAAAAAALAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
Do you know if we already have the functionality to generate the address from the chunk's entry in the data map?  We will have to remember to modify this function to get the address that way, once we have the ability to do so.  

@pandora23  Do we already have methods yet that will generate the 81 character tryte address from a chunk's hash in the data map?